examples/portworx/variables.tf

```diff
@@ -7,23 +7,23 @@ variable "install_storage" {
   description = "If set to false does not install storage and attach the volumes to the worker nodes. Enabled by default"
 }
 
-variable unique_id {
+variable "unique_id" {
```

Comment:
Do this file need to be removed as its for portworx?

Reply:
This one is correct as it is in the examples/portworx/variables.tf file.  It should be removed from the examples/odf/variables.tf file